http/oidc/src/main/java/org/wildfly/security/http/oidc/ElytronMessages.java

```diff
@@ -287,5 +287,17 @@ interface ElytronMessages extends BasicLogger {
 
     @Message(id = 23073, value = "Nonce cookie does not exist")
     String nonceCookieDoesNotExist();
+
+    @Message(id = 23074, value = "Invalid logout output: %s is not a valid value for %s")
+    RuntimeException invalidLogoutPath(String pathValue, String pathName);
+
+    @Message(id = 23075, value = "Invalid %s: %s the value must be an absolute URI")
```

Comment:
s/the value/the logout callback path value

Reply:
done